app/services/proofing/socure/id_plus/response.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Socure
+    module IdPlus
+      class Response
+        # @param [Faraday::Response] http_response
+        def initialize(http_response)
+          @http_response = http_response
+        end
+
+        # @return [Hash<Symbol,Boolean>]
+        def kyc_field_validations
+          @field_validations ||= kyc('fieldValidations').
+            each_with_object({}) do |(field, valid), obj|
+              obj[field.to_sym] = valid.round == 1
```

Comment:
would we ever want this logic to change per-field? or are we set with anything above `0.5` is a pass

Reply:
They will literally only send either `0.99` or `0.01`, I just figured `.round` would avoid any weirdness with `==` and floating points